app/src/main/java/com/netguru/codereview/MainActivity.kt

```diff
@@ -1,11 +1,14 @@
 package com.netguru.codereview
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.netguru.codereview.ui.MainFragment
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        supportFragmentManager.beginTransaction().replace(R.id.container, MainFragment()).commitNow()
```

Comment:
**Warning**
[Transactions committed in this way may not be added to the FragmentManager's back stack, as doing so would break other expected ordering guarantees for other asynchronously committed transactions. This method will throw `IllegalStateException` if the transaction previously requested to be added to the back stack with `addToBackStack`.](https://developer.android.com/reference/androidx/fragment/app/FragmentTransaction#:~:text=Transactions%20committed%20in,stack%20with%20addToBackStack)